### ISSUE TITLE
TypeError: (0 , _jestSnapshotsSvg.addFontFallback) is not a function

### ISSUE THREAD

**Issue by jsdario** [jest-community/jest-snapshots-svg#42]
I am creating a list component with text inside. Since I didn't see the text in it, I tried to add a font fallback to see if that was the problem. Then this error prompted.

Calling:
```
import { addFontFallback } from 'jest-snapshots-svg'
...
addFontFallback('Proxima Nova', "'Open Sans', sans-serif")
addFontFallback('Proxima Nova Alt', "'Open Sans', sans-serif")
```

With `"jest-snapshots-svg": "^0.0.24"`

Any insights? Should / Can I help to fix this? I did some inspection, but nothing seemed weird to me.

**Comment by jacobp100** [comment]
I think we just need to publish. @orta

In the mean time, try `npm install https://github.com/jest-community/jest-snapshots-svg.git`.

**Comment by dawnmist** [comment]
Released in 0.1.0.
